pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -192,3 +192,7 @@ func (c *Compiler) stepWorkdir(container *yaml.Container) string {
 	}
 	return filepath.Join(c.base, c.path, container.Directory)
 }
+
+func isPlugin(container *yaml.Container) bool {
+	return len(container.Commands) < 1 && len(container.Command) < 1
```

Comment:
This is more readable imho
```suggestion
	return len(container.Commands) == 0 && len(container.Command) == 0
```